packages/otelbin/src/app/og/ParentsNode.tsx

```diff
@@ -0,0 +1,180 @@
+// SPDX-FileCopyrightText: 2023 Dash0 Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import React from "react";
+import { type Node } from "reactflow";
+import ParentNodeTag from "./ParentNodeTag";
+import ArrowRight from "./svg/move-right.svg";
+import { ReceiversNode, ProcessorsNode, ExportersNode } from "./NodeTypes";
+import Workflow from "./svg/workflow.svg";
+import BarChart4 from "./svg/bar-chart-4.svg";
+import ListTree from "./svg/list-tree.svg";
+
+export const parentNodesConfig = [
```

Comment:
**Issue:** This whole code block is duplicated.

Reply:
Some code from the client-side is not supported in the Open Graph (og) route on the server-side in the usual way. I will investigate further.

Reply:
Fixed it and removed the duplicates